app/src/main/java/com/edulectronics/tinycircuit/Models/MessageArgs.java

```diff
@@ -9,20 +9,19 @@
 public class MessageArgs implements Serializable{
     public MessageTypes type;
     public int title;
-    public int message;
+    public String message;
 
     public boolean endActivity;
     public boolean goToNextScenario;
 
-    public MessageArgs(int message, MessageTypes type) {
+    public MessageArgs(String message, MessageTypes type) {
```

Comment:
Waarom is dit een string? Eigenlijk wil je alleen een resource ID kunnen gebruiken, anders stel je iemand in de gelegenheid slechte code te schrijven door een hardcoded string erin te gooien.